spring-core/pom.xml

```diff
@@ -50,6 +50,11 @@
             <artifactId>javax.inject</artifactId>
             <version>1</version>
         </dependency>
+        <dependency>
```

Comment:
There is junit version 4.11 already in this pom. I believe you don't need yet another junit